inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/StreamSinkClient.java

```diff
@@ -87,6 +97,22 @@ public Pair<Boolean, String> updateSink(SinkRequest sinkRequest) {
         }
     }
 
+    /**
+     * Update the stream sink by key
+     * @param sinkRequest
+     * @return
```

Comment:
If no description for `param` and `return`, just remove them.

Reply:
fixed, thx